src/main/scala/tech/cryptonomic/conseil/metadata/MetadataService.scala

```diff
@@ -43,13 +43,13 @@ class MetadataService(
         entities.map(entity => networkPath.addLevel(entity.name))
     }.toSet
 
-    val result = networkPaths.map { path =>
+    val result = Future.traverse(networkPaths){ path =>
```

Comment:
Do I understand correctly that this change makes the code more parallel, i.e. multiple futures are spawned together instead of being spawned sequentially?

Reply:
In fact it's a side-effect of switching from `map` and then `sequence`, I didn't realise the fact.
I did it out of habit, because doing both operations is always equivalent to a single `traverse` and I refactor it without thinking to make the intent explicit.

In this case we might've gained a little performance too, possibly, but it wasn't intentional on my part.